docs/benchmarks.md

```diff
@@ -5,57 +5,66 @@
-* OS: debian-11
-* VM location: us-west1-c
+* VM Type: n2-standard-96
+* OS:  ubuntu-20.04
+* VM Bandwidth: 100Gbps
+* VM location: us-west1-b
+* Disk Type: SSD persistent disk
 * GCS Bucket location: us-west1
 * Framework: FIO
 
 ### FIO Spec
-* Test runtime: 60sec
-* Thread count: 40
-* Block Size: 256KB for 256KB files and 1MB for all other files.
-
-### GCSFuse command
+* FIO creates a number of threads or processes doing a particular type of I/O action specified by the user.
+* The use of fio is to write a job file matching the I/O load one wants to simulate.
+* For testing different sizes of the file, change file size parameters.
+* BlockSize defines how large size we are issuing for I/O. For 256kb we used 16k block size other than that we used 1M blocksize.
+* We have a fsync parameter for writes that defines fio will sync the file after every fsync number of writes issued. When the Writefile operation called GCSFuse will write the data to disk. When the SyncFile operation called GCSFuse will write the data from disk to GCS bucket. So after fsync write operation GCSFuse will write the data into the GCS bucket.
 ```
-gcsfuse --implicit-dirs --stat-cache-ttl=60s --type-cache-ttl=60s 
---client-protocol=http1 --max-conns-per-host=100 <bucket-name> <path-to-mount-point>
+gcsfuse --implicit-dirs  --client-protocol=http1 --max-conns-per-host=100 <bucket-name> <path-to-mount-point>
 ```
-## Reads
-### Sequential reads
 
-| File Size | Bandwidth in GiB/sec | IOPS | Avg Latency (msec) |
-|-----------|----------------------|------|--------------------|
-| 256KB     | 0.47                 | 1875 | 1350               |
-| 1MB       | 1.56                 | 1554 | 1623               |
-| 2MB       | 2.27                 | 2279 | 1110               |
-| 5MB       | 3.75                 | 3796 | 670                |
-| 10MB      | 4.02                 | 4071 | 625                |
-| 50MB      | 4.37                 | 4433 | 574                |
-| 100MB     | 4.33                 | 4390 | 579.94             |
-| 200MB     | 4.31                 | 4371 | 582.89             |
-| 1GB       | 4.32                 | 4383 | 580.67             |
+### Sequential Write
+
+| File Size | Bandwidth in MiB/sec   | IOPS(avg) | Avg Latency (msec)   |
+|-----------|------------------------|-----------|----------------------|
+| 256KB     | 62.3                   | 9872.44   | 2.278                |
+| 1MB       | 2524                   | 3871.71   | 15.150               |
+| 50MB      | 3025                   | 4588.38   | 19.991               |
+| 100MB     | 2904                   | 6242.30   | 18.648               |
+| 1GB       | 2875                   | 11155.96  | 9.789                |
+| 4GB       | 477                    | 850.86    | 175.337              |
 
-### Random reads
+### Random Write
+In the case of Random writes, only offset will change; there are no changes in gcs calls, for that reason bandwidth and other results remain the same.
```

Comment:
please dont use capital case inbetween the statement.

In case of random writes, only offset will change in calls issued by fio. GCSFuse behaviour will remain the same and there are no changes in the way gcs calls are being made. Hence the bandwidth will be same as sequential reads.